arch/x86/x86_64/include/uk/asm/arch.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+#ifndef __UK_ARCH_H__
```

Comment:
I think that header guard is a bit too ambiguous and might cause problems.

Reply:
Good point, I'll change it to `__UK_ARCH_X86_64_H`